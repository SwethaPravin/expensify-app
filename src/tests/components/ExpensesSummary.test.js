import React from 'react';
import { shallow } from 'enzyme';
import {ExpensesSummary} from '../../components/ExpensesSummary';
import toJSON from 'enzyme-to-json';


test('should render expenses summary with 1 expenses', () =>{
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal ={9400}/>);
  expect(toJSON(wrapper)).toMatchSnapshot();
});

test('should render expenses summary with 2 expenses', () =>{
  const wrapper = shallow(<ExpensesSummary  expenseCount={2} expensesTotal ={9400}/>);
  expect(toJSON(wrapper)).toMatchSnapshot();
});