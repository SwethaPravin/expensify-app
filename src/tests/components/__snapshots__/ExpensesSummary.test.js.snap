// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expenses summary with 1 expenses 1`] = `
<div>
  <h1>
     Viewing 
    1
     
    expense
     totalling 
    $94.00
     
  </h1>
</div>
`;

exports[`should render expenses summary with 2 expenses 1`] = `
<div>
  <h1>
     Viewing 
    2
     
    expenses
     totalling 
    $94.00
     
  </h1>
</div>
`;
